packages/client/README.md

```diff
@@ -10,97 +10,188 @@
 
-The client is available on [npm](https://www.npmjs.com/package/streamr-client) and can be installed simply by:
+```js
+const streamr = new StreamrClient({
+    auth: {
+        privateKey: 'your-private-key'
+    }
+})
+```
 
+#### Creating a stream and publishing data to it
+```js 
+// Requires MATIC tokens (Polygon blockchain gas token)
+const stream = await streamr.createStream({
+    id: '/foo/bar'
+})
+
+await stream.publish({ timestamp: Date.now() })
 ```
-npm install streamr-client
+
+#### Subscribing
+```js 
+const streamId = 'binance-streamr.eth/ETHUSDT/ticker'
+
+streamr.subscribe(streamId, (message) => {
+    // handle for individual messages
+})
+
 ```
 
-## Usage
+More examples can be found in the [examples repo](https://github.com/streamr-dev/examples).
+___
 
-Here are some usage examples. More examples can be found [here](https://github.com/streamr-dev/examples).
 
-In Streamr, Ethereum accounts are used for identity. You can generate an Ethereum private key using any Ethereum wallet, or you can use the utility function `StreamrClient.generateEthereumAccount()`, which returns the address and private key of a fresh Ethereum account.
+## Setup
 
-### Creating a StreamrClient instance
+### Installation
+The client is available on [npm](https://www.npmjs.com/package/streamr-client) and can be installed simply by:
 
-```js
-const client = new StreamrClient({
-    auth: {
-        privateKey: 'your-ethereum-private-key'
-    }
-})
+```
+npm install streamr-client
 ```
 
-When using Node.js remember to import the library with:
+### Importing `streamr-client`
+To use with react please see [streamr-client-react](https://github.com/streamr-dev/streamr-client-react)
 
+If using typescript you can import the library with:
 ```js
 import { StreamrClient } from 'streamr-client'
 ```
+When using Node.js you can import the library with:
 
-### Subscribing to real-time events in a stream
+```js
+const { StreamrClient } = require('streamr-client')
+```
+
+For usage in the browser include the latest build, e.g. by including a `<script>` tag pointing at a CDN:
+
+```html
+<!-- for Brubeck package (6x) -->
+<script src="https://unpkg.com/streamr-client@beta/streamr-client.web.js"></script>
+```
+___
+## Usage
+### Client Creation
+If you don't have an Ethereum account you can use the utility function [StreamrClient.generateEthereumAccount()](#utility-functions), which returns the address and private key of a fresh Ethereum account.
+
+More information on Stream IDs under the [stream creation project docs](https://streamr.network/docs/streams/creating-streams)
```

Comment:
Maybe this could be stated a bit later?

Reply:
Or as user maybe doesn't need authentication at all, we could leave out the paragraph here. If he wants a specific authentication, we surely has an Ethereum key already (and therefore doesn't need the utility function).